Gruntfile.js

```diff
@@ -69,19 +69,46 @@ module.exports = function (grunt) {
                 },
                 src: ['test/testFrameworks/mocha-k3po*.js']
             },
-            testMochaK3poBrowserSupport: {
+            testMochaK3poLocalBrowserSupport: {
                 options: {
                     reporter: 'spec',
                     ui: 'mocha-k3po',
                     require: 'lib/testFrameworks/mocha-k3po.js',
                     captureFile: "build/testMochaK3po.txt",
+                    // timeout: 5000,
                     browser: {
+                        // debug: true,
                         desiredCapabilities: {
                             browserName: 'firefox'
                         }
                     }
                 },
                 src: ['test/testFrameworks/mocha-k3po*.js']
+            },
+            testMochaK3poSauceBrowserSupport: {
+                options: {
+                    reporter: 'spec',
+                    ui: 'mocha-k3po',
+                    require: 'lib/testFrameworks/mocha-k3po.js',
+                    captureFile: "build/testMochaK3po.txt",
```

Comment:
from what I read it captures all the output of from the test run(I left it there because it was in the task to run tests locally, on which I based the sauceTest task)